### PR TITLE
[4.0] mail templates empty style

### PR DESCRIPTION
**Pull request by brianteeman**
This PR removes an empty inline style. It can never have a value afaict and we are supposed to be moving away from using any inline styles for CSP etc

Pull Request for Issue #26476

### CODE REVIEW

**Comment by brianteeman** [comment]
drone failures are the usual false failures

File: administrator/components/com_mails/tmpl/templates/default.php [hunk 1]
```diff
@@ -65,8 +65,7 @@
 								<td class="d-md-table-cell">
 									<?php foreach ($this->languages as $language) : ?>
 										<?php $exists = in_array($language->lang_code, $item->languages); ?>
```
Comment (Quy) [inline administrator/components/com_mails/tmpl/templates/default.php]: Remove variable/line?

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on c5317ecb3b7e24a640887a42bd36d90a6c6e22f7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27326">issues.joomla.org/tracker/joomla-cms/27326</a>.</sub>

**Comment by 810** [comment]
I have tested this item :white_check_mark: successfully on c5317ecb3b7e24a640887a42bd36d90a6c6e22f7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27326">issues.joomla.org/tracker/joomla-cms/27326</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/27326">issues.joomla.org/tracker/joomla-cms/27326</a>.</sub>

**Comment by brianteeman** [comment]
thanks

**Pull request merged**